### ISSUE TITLE
Dynamic Child of, Exclude term etc. fileds of Listing Grid

### ISSUE THREAD

**Issue by logobby** [Crocoblock/suggestions#652]
Hi all,

What do you think to make the queries in the Listing Grid widget more dynamically?
For example I want to display in an archive page, the child categories of the current parent category...
I can do it by inserting the id of the parent category in the 'Child of' input but it's still "static" and won't work for another categories...so I want it to be dynamic like to choose "Child of: Current category" or other dynamic params...instead of creating several template for each category.

A least if it was possible to insert shortcode in the fields it would have been fantastic!
 
Please vote fo this feature ans maybe we will see it in the next release :)

![qbs6bGu4BRMUs9jaUDmU6pYmHjF1laBQwQ](https://user-images.githubusercontent.com/40466413/68544071-cb281b80-03c7-11ea-88cd-0f6d28ea6112.png)
 
Thanks
Sam

**Comment by logobby** [comment]
Hi all,

Finally I just remembered the option to adding a %current_id% (for example) and it's working (I used this in other places in the website but forgot :) ) and when I asked to the support, they suggested me to open a feature request on Github so I was sure it's not possible :)

But after that I noticed that I have another request...maybe someone will have a good solution for that:
I have Categories and Subcategories ok for a cpt?
On the home page I display the parent categories.
On the archive page category I display the videos (it's a custom cpt) + the subcategories of the current category like blocks at the top, and I'm using the %current_id% in the 'child of' input and it's work, but when I enter to a subcategory, the subcategories blocks disapear and it's normal because they don't have subcategories :)
So now, I wish to display there, the subcategories of the parent category.

Example:
Category A have subcategories A1 A2 etc.
I enter to A - I can see A1 A2 etc.
I enter to A1 - I wish to still see the the A1 A2 subcategories blocks.

To achieve that the solution I have now it's to create for each parent category an archive template and insert the id of the category (for example id 6) and it's work like a charm also for the subcategories (with the according display conditions).

But I wish to make it with only one archive if it's possible, maybe not, I'm trying to find a solution...maybe duplicate the widget, add an id and via the code apply a solution...

If you have any suggestion other I'll be happy to hear 😊 thanks
